### DISCUSSION TITLE
Wyze alternative firmware

### DISCUSSION THREAD

**Discussion by alan5ab** [mrlt8/docker-wyze-bridge#1356] [General]
I  recently found some open source firmware that removes your wyze cam from from the wyze eco-system. I have it running on 3 -V3s presently, search "thingino" on github. FYI

**Comment by g3rmw0rm** [comment]
any more information you can share on this by chance?

**Comment by alan5ab** [comment]
You guys having problems should look into this, I have no regrets leaving the wyze ecosystem.
https://thingino.com/

**Reply by cheme75** [reply]
> You guys having problems should look into this, I have no regrets leaving the wyze ecosystem.
> 
> https://thingino.com/

Last I looked it was pretty limited. How to load on cams without sd?

**Reply by alan5ab** [reply]
It's for ingenic processors, what wyze camera doesn't have an sd card slot. I have used uart to load it on chinese poe cams with no usb/sd card

**Reply by cheme75** [reply]
> It's for ingenic processors, what wyze camera doesn't have an sd card slot. I have used uart to load it on chinese poe cams with no usb/sd card

V1 db cam has no sd. Just checked wiki info and it says can load via usb. I got a couple beta db cams still so maybe I’ll try one as a test. Not what otg usb is so another thing to look up.

I tried to load Ubuntu to run dwb in VYM on windows but pc seems to struggle and interface for Ubuntu sucks between partially functional basic views and enhanced views, it is incredibly frustrating. Managed to get dwb running but now it doesn’t like the dash character in the Wyze api ID. It’s down to the one Wyze db cam so not a huge deal. Worst case is I can just use via app until it craps out. Got a couple of v2 cams but not using them.

**Reply by alan5ab** [reply]
I was going to order a Wdb1 since they're so cheap on ebay but I already got too many cameras. There's a few guys who really like them. The wyze V2 are ok as long as your not looking for night vision

**Reply by java007md** [reply]
> It's for ingenic processors, what wyze camera doesn't have an sd card slot. I have used uart to load it on chinese poe cams with no usb/sd card

Can you share what specific Chinese POE Cams you have thingino loaded on?

I'm also pretty happy with Thingino firmware testing I've done on a couple of Wyze V3 cams.

**Reply by alan5ab** [reply]
I got the t31x/gc4653 it's 4mp but the night vision hasn't impressed me at all. https://www.aliexpress.us/item/3256807871057604.html
Really prob the best day/night camera I have is the reolink CX410, it blows the Wyze V3 away after dark

**Reply by wcarrx** [reply]
Tried Thingino on a  Cam V3 -picture display was terrible (streaming through VLC), could not find any settings that fixed it. As far as restoring to Wyze software: camera is bricked.  It will only boot to Thingino now. Tried all possible recovery routes except opening the camera and accessing it's UART.

**Reply by alan5ab** [reply]
Look thru their wiki, you can install go2rtc on an sd card and run on camera. I have put it on the cameras I've had problems with and it helps. Browse their wiki for $15 aliexpress NVR too. I have 3 of them and they work good with any onvif or rtsp stream.
https://github.com/themactep/thingino-firmware/wiki/go2rtc
https://github.com/themactep/thingino-firmware/wiki/Integration:-$15-NVR

**Reply by wcarrx** [reply]
Thanks -- I'll take a look
